packages/TypeDeclaration/tests/Rector/Property/PropertyTypeDeclarationRector/Fixture/typed_redax.php.inc

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Rector\TypeDeclaration\Tests\Rector\FunctionLike\PropertyTypeDeclarationRector\Fixture;
+
+class TypedRedax
```

Comment:
Nit.. the tool is named „redaxo“
```suggestion
class TypedRedaxo
```

Reply:
I didn't notice that. It was just work in progress, untill I found the problem.
Thank for pointing it out, I give me more reason to think about better naming.

Got descriptive name now :)